microsoft-edge/devtools-guide-chromium/evaluate-performance/index.md

```diff
@@ -112,15 +112,11 @@ Once you have a recording of the page's performance, you can assess the page's p
 
 The main metric for measuring the performance of any animation is frames per second (FPS).  Users are happy when animations run at 60 FPS.
 
-1. Look at the **FPS** chart, shown below.  Whenever a red bar is displayed above **FPS**, it means that the framerate dropped so low that it's probably harming the user experience.  In general, the higher the green bar, the higher the FPS.
-
-   ![The FPS chart.](../media/evaluate-performance-performance-fps-chart.msft.png)
```

Comment:
Before we merge this, can you check if this image is used anywhere else in the docs? If not, then please delete the image file. Thanks!